macro/makeGenieEvents.py

```diff
@@ -44,6 +44,7 @@ def get_arguments(): #available options
   ap.add_argument('-n', '--nevents', type=int, help="number of events", dest='nevents', default=100)
   ap.add_argument('-e', '--event-generator-list', type=str, help="event generator list", dest='evtype', default=None) # Possbile evtypes: CC, CCDIS, CCQE, CharmCCDIS, RES, CCRES, see other evtypes in $GENIE/config/EventGeneratorListAssembler.xml
   ap.add_argument("--nudet", dest="nudet", help="option for neutrino detector", required=False, action="store_true")
+  ap.add_argument('--MS', help="make a new spline file", dest='MakeSpline', action="store_true")
```

Comment:
I would make make the option longer easier to understand, e.g. `--make_spline` or something like that. If you want a short option as well, that could be e.g. `-s`.

Reply:
Ok, I have renamed the option as suggested. Since -s is already used for the seed, as a short alternative I have set -ms.
Thank you for your comment.